### ISSUE TITLE
When Ntopng is restarted, all counters reseted

### ISSUE THREAD

**Issue by hany-mamdoh** [ntop/ntopng#1754]
When Ntopng is restarted, all counters get reset and Ntop traffic stats page starts showing the output from the beginning

I need to have a feature so that recorded on harddisk and when the service restarted or pfsense restarted then it continue and not to start from beginning

**Comment by emanuele-f** [comment]
Also related to https://github.com/ntop/ntopng/issues/1054

**Comment by hany-mamdoh** [comment]
I need it to be accumulated and all be saved on hard disk so when restart done all remains not to start from beginning 
thanks for your effort

**Comment by simonemainardi** [comment]
You should consider the following things to do what you've asked.

1. Tune local hosts cache duration

![image](https://user-images.githubusercontent.com/3397663/43588491-e69a821e-966c-11e8-895c-e5ed4c0f6c4f.png)

Counters of your local hosts that become idle are dumped to a redis cache. By default, a dump lasts for 1 hour. This means that:
- if an idle local host becomes active again in less than 1 hour, its counters are restored from redis
- if an idle local host becomes active again after more than 1 hour, its counters are reset and they will start from zero (counters are reset, timeseries will be untouched)

So, if you want to keep dumps for more than 1 hour, you just have to tune the preference indicated above.

2. Make your hosts sticky

```
[--sticky-hosts|-S] <mode>          | Don't flush hosts (default: none).
                                    | Values:
                                    | all    - Keep all hosts in memory
                                    | local  - Keep only local hosts
                                    | remote - Keep only remote hosts
                                    | none   - Flush hosts when idle
```

You can also prevent ntopng from purging idle hosts from memory if you use option `--sticky-hosts`. This means that an host will always stay in memory and its counters won't never be reset. The only thing to keep in mind here is when ntopng stops and restarts:

- when ntopng stops, it will dump all the local host counters as it always does
- when ntopng restarts, if sticky hosts are enabled, it will immediately reload all the dumped hosts in memory thus restoring their counters (if dumps are not expired according to 1).

So basically you can combine 2. and 1. to obtain the desired behavior
